### PR TITLE
azure_stress_net: Added pipeline definition

### PR DESCRIPTION
**Pull request by ader1990**

### CODE REVIEW

File: linux_pipeline/Jenkinsfile_azure_stress_networking_watcher [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env groovy
+
+properties ([
+    pipelineTriggers([cron('H 1 * * *')])
+])
+
+env.WORK_DIR = "/mnt/tmp/watcher-azure-stress-networking"
+env.IMAGE_TAGS = "Ubuntu 18.04-LTS"
+env.PIPELINE_NAME = "pipeline-azure-stress-networking"
```
Comment (juhlee-microsoft) [inline linux_pipeline/Jenkinsfile_azure_stress_networking_watcher]: @ader1990 This is about new image monitoring in Azure Market. What makes you pick up 'stress' in pipeline name?
Can you choose something else like 'monitor', 'sanity-check', etc?

Comment (ader1990) [reply linux_pipeline/Jenkinsfile_azure_stress_networking_watcher]: that was the initial test case, I will change the name to watcher-azure-latest-image

Comment (juhlee-microsoft) [reply linux_pipeline/Jenkinsfile_azure_stress_networking_watcher]: Thanks . I like it.

File: linux_pipeline/Jenkinsfile_azure_latest_image [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            name: 'VM_SIZE',
+            defaultValue: "Standard_D14_v2",
+            description: 'Azure vm size to be tested. The vm size requires support for 8 nics.'],
+        [$class: 'StringParameterDefinition',
+            name: 'TEST_CASE',
+            defaultValue: "STRESSTEST-VERIFY-RESTART-NETWORK,VERIFY-IFUP-8-NICS-SRIOV",
+            description: 'Test cases to be run, comma separated'],
+        [$class: 'StringParameterDefinition',
+            name: 'TEST_ITERATIONS',
+            defaultValue: "3",
```
Comment (ader1990) [inline linux_pipeline/Jenkinsfile_azure_latest_image]: the default value should be 1000 iterations

File: linux_pipeline/Jenkinsfile_azure_latest_image [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            description: 'How many vms to spawn for each test']
+        ]
+    ]
+])
+
+def gitRemoteUrl = "https://github.com/ader1990/LISAv2.git"
```
Comment (ader1990) [inline linux_pipeline/Jenkinsfile_azure_latest_image]: this needs to be changed once the lisav2 patch is merged

File: linux_pipeline/Jenkinsfile_azure_latest_image_watcher [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+            archiveArtifacts env.LATEST_IMAGES
+            currentBuildImages.each() {
+                def image = it.replace("\n", "").replace("\r", "");
+                if (image && !previousImagesResult.contains(image)) {
+                    runs["${image}"] = {
+                        build (job: "${env.PIPELINE_NAME}/${env.BRANCH_NAME}",
+                            parameters: [
+                                string(name: 'AZURE_IMAGES', value: "${image}"),
+                                string(name: 'AZURE_REGIONS', value: "westus2"),
+                                string(name: 'TEST_ITERATIONS', value: "1"),
```
Comment (ader1990) [inline linux_pipeline/Jenkinsfile_azure_latest_image_watcher]: test iterations should be 1000

**Pull request merged**